Shared/Samples/Identify features in WMS layer/README.metadata.json

```
@@ -0,0 +1,30 @@
+{
+    "category": "Layers",
+    "description": "Identify features in a WMS layer and display the associated popup content.",
+    "ignore": false,
+    "images": ["identify-features-in-wms-layer.png"],
```

```suggestion
    "images": [
        "identify-features-in-wms-layer.png"
    ],
```

See [this comment](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/421#discussion_r1614158578).